src/main/java/frc/robot/Intake.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * Add&named motors in Intake.
+ */
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);
+    private Solenoid piston = new Solenoid(Ports.Intake.SOLENOID);
+
+    /**
+     * Add motor inverted.
+     * Add data of is Motor inverted
+     */
+    public Intake() {
+        motor.setInverted(Ports.Intake.IS_MOTOR_INVERTED);
+    }
+
+    /**
+     * Set percentage for motor.
+     * @param power the speed times the torque.
+     */
+    public void powerWheels(double power) {
+        motor.set(ControlMode.PercentOutput, power);
+    }
+
+    /**
+     * A check to see if piston is open or close, true= open, false= close.
+     * @return
+     */
+    public boolean isPistonEngaged() {
+        return piston.get();
+    }
+
+    /**
+     * Set piston position.
+     * @param engaged
+     */
+    public void setPistonMode(boolean engaged) {
+        piston.set(engaged);
+    }
+
+    /**
+     * Change piston position.
+     */
+    public void toggle() {
+        if (isPistonEngaged()) {
+            setPistonMode(false);
+        } else {
+            setPistonMode(true);
+        }
+
+    }
+
+}
+
+
+
+
```

Comment:
```suggestion
    }
}

```
No need for all these blank lines 🙂